src/strategies_and_params/replicastrategy.jl

```diff
@@ -76,17 +75,18 @@ The `r{i}s{k}_dot_r{j}s{k}` overlap can be omitted with the flag `vecnorm=false`
 
 By default, overlaps of different spectral states are omitted. To include overlaps of
 different spectral states `r{i}s{k}_dot_r{j}s{l}` and `r{i}s{k}_Op{m}_r{j}s{l}`, use the
-flag `mixed_spectral_overlaps=true`. 
+flag `mixed_spectral_overlaps=true`.
 
 See [`ProjectorMonteCarloProblem`](@ref), [`ReplicaStrategy`](@ref) and
 [`AbstractOperator`](@ref Interfaces.AbstractOperator) (for an interface for implementing
 operators).
 
 # Transformed Hamiltonians
 
-If a transformed Hamiltonian `G` has been passed to [`ProjectorMonteCarloProblem`](@ref)
-then overlaps can be calculated by passing the same transformed Hamiltonian to `AllOverlaps`
-by setting `transform=G`. A warning is given if these two Hamiltonians do not match.
+If a transformed Hamiltonian `G` has been passed to [`ProjectorMonteCarloProblem`](@ref), an
+inverse transformation is applied to the operators in `AllOverlaps`. Additinally, an
+operator representing the inverse transform applied to the identity operator is added to
+the list of operators.
```

Comment:
```suggestion
inverse transformation is applied to the operators in `AllOverlaps`. Additionally, an
operator representing the inverse transform applied to the identity operator is added to
the list of operators.
```

Also, should the docstring say that the keyword argument `transform` is now deprecated and has no effect?